libraries/cyclestreets-core/src/main/java/net/cyclestreets/api/client/geojson/MapsFactory.kt

```diff
@@ -0,0 +1,35 @@
+package net.cyclestreets.api.client.geojson
+
+import net.cyclestreets.api.Map
+import net.cyclestreets.api.Maps
+import net.cyclestreets.api.client.geojson.AbstractObjectFactory.propertyOrDefault
+import org.geojson.Feature
+import org.geojson.FeatureCollection
+
+class MapsFactory {
+    companion object {
+        fun toMaps(featureCollection: FeatureCollection): Maps {
+            return Maps(
+                featureCollection.features
+                        .map { f -> toMap(f) }
+                        .filter { m -> isBritainOrIreland(m) }
```

Comment:
I agree with this restriction as a starting point.  We can consider adding others subsequently as part of #46.